src/routes/home/index.js

```diff
@@ -11,20 +11,13 @@ import React from 'react';
 import Home from './Home';
 import Layout from '../../components/Layout';
 
-async function action({ fetch }) {
-  const resp = await fetch('/graphql', {
```

Comment:
Nit: Since this fetches the /graphql endpoint of the app with a query for a new article, it might make sense to also delete the GraphQL code that serves this API response.